vcf-dumper/vcf-dumper-lib/src/main/java/uk/ac/ebi/eva/vcfdump/BiodataVariantToVariantContextConverter.java

```diff
@@ -79,6 +86,43 @@ public VariantContext transform(Variant variant) {
         return variantContext;
     }
 
+    private String getAnnotationAttributes(Variant variant) {
+        List<ConsequenceType> consequenceTypes = variant.getAnnotation().getConsequenceTypes();
+        String allele = variant.getAnnotation().getAlternativeAllele();
+        String csq = "";
```

Comment:
This is a matter of taste and there may be different opinions, but, I like more initializing csq to null, and then check it later csq != null instead of !csq.isEmpty(). If csq is null and you call later to some method in the csq object, you would  get an exception, but if csq is an empty string, you wouldn't, so some errors could be masked and more difficult to detect